modules/platforms/cpp/ignite/client/detail/sql/sql_impl.cpp

```diff
@@ -15,11 +15,12 @@
  * limitations under the License.
  */
 
-#include "ignite/client/detail/sql/sql_impl.h"
+#include "sql_impl.h"
+
 #include "ignite/client/detail/sql/result_set_impl.h"
 #include "ignite/client/detail/utils.h"
 
-#include "ignite/schema/binary_tuple_builder.h"
+#include <ignite/tuple/binary_tuple_builder.h>
```

Comment:
Why `<>` instead of `""`?

Reply:
In this case there is no difference, but I prefer to use the same pattern for all #include directives.

If we have a public header installed e.g. in /usr/local/include/ignite and this header includes another public header, the user can compile a client app with -I/usr/local/include compiler option then it will find all the headers.

So if you include a header that starts with ignite/... I prefer to have it always in angle brackets.

But if you include a header in the same subdir you can use relative paths in quotes and omitting the common prefix.